util/Hexagon/README.md

```diff
@@ -0,0 +1,57 @@
+# Hexagon/Halide Instrumentation
+
+Adds timing and reporting to Hexagon code generated by a Halide Generator.
+
+## Input Files
+`instrumented_generator.h`
+`profiling.cpp`
+`logcat2json.cpp`
+
+## Building
+For now, build `profiling.cpp` with your DSP code. Later we will likely include this precompiled binary.
+
+```
+	hexagon-clang++ -std=c++17 -Igenerated -fPIC profiling.cpp -c -o profiling.o
+```
+
+Then link `profiling.o` with `lib${MY_LIB}_skel.so`.
+
+To build `logcat2json`, from the source root directory:
+```
+    mkdir build
+    cmake -B ./build -S . && cmake --build ./build
+```
+
+## Profiling
+1. Include the header `instrumented_generator.h` in your Halide generator source.
```

Comment:
This isn't getting installed, so how are users supposed to get this header?

Reply:
I'm not sure. I have tried adding an instrumented_generator target, I think this should do it.

Reply:
This is going to require a change in `/packaging/CMakeLists.txt` to install the `logcat2json` tool and the `InstrumentedGenerator` target. The latter should be grouped with the other platform-agnostic (i.e. source-only) targets.

Reply:
The `reports/*.py` scripts should also be installed somewhere beneath `share/`, maybe near the autoscheduling helper scripts? When installed, we can add symlinks without the `.py` extension, if you like.